src/app/components/question-create/question-create.component.html

```diff
@@ -0,0 +1,119 @@
+<div class="wrapper">
+  <p class="title">
+    Let`s create a question!
+  </p>
+
+  <form [formGroup]="createQuestionForm" (ngSubmit)="onSubmit()" class="question-form">
+    <mat-form-field class="question-input-field" appearance="fill">
+      <textarea
+        formControlName="question"
+        matInput
+        placeholder="What`s the difference between var, const and let?"
+      ></textarea>
+    </mat-form-field>
+
+    <mat-radio-group
+      formControlName="typeOfQuestion"
+      class="radio-buttons"
+      aria-label="Select an option"
+      (input)="chooseTypeOfQuestion($event)"
+    >
+      <mat-radio-button class="radio-button" value="Single">Single Choice</mat-radio-button>
+      <mat-radio-button class="radio-button" value="Multiple">Multiple Choice</mat-radio-button>
+      <mat-radio-button class="radio-button" value="Open">Open</mat-radio-button>
+    </mat-radio-group>
+
+    <div *ngIf="createQuestionForm.controls['typeOfQuestion'].value === 'Single'" class="container-single-choice">
```

Comment:
better to create enum TypeOfQuestion and add getters in this component like isSingle, looks to hard when we compare it in html